src/mixins/I18n.ts

```diff
@@ -62,11 +62,15 @@ export interface I18nMixin {
 	 * The localized messages, along with a `format` method for formatting ICU-formatted templates.
 	 */
 	localizeBundle<T extends Messages>(bundle: Bundle<T>): LocalizedMessages<T>;
+
+	properties: I18nProperties;
 }
 
-export function I18nMixin<T extends Constructor<WidgetBase<I18nProperties>>>(base: T): T & Constructor<I18nMixin> {
+export function I18nMixin<T extends Constructor<WidgetBase<any>>>(base: T): T & Constructor<I18nMixin> {
```

Comment:
Does the default screw things up here?  I just always hate `any`, it feels dirty...